Index: app/src/main/java/com/yum/two_yum/TestActivity.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/yum/two_yum/TestActivity.java	(date 1522682244000)
+++ app/src/main/java/com/yum/two_yum/TestActivity.java	(revision )
@@ -6,7 +6,5 @@
  */
 
 public class TestActivity {
-dsds
-22
 
 }
